libs/auth/src/certs.ts

```diff
@@ -16,9 +16,9 @@ const VX_IANA_ENTERPRISE_OID = '1.3.6.1.4.1.59817';
 const VX_CUSTOM_CERT_FIELD = {
   /** One of: card, admin, central-scan, mark, scan (the latter four referring to machines) */
   COMPONENT: `${VX_IANA_ENTERPRISE_OID}.1`,
-  /** Format: {state-2-letter-abbreviation}.{county-or-town} (e.g. MS.Warren) */
+  /** Format: {state-2-letter-abbreviation}.{county-or-town} (e.g. ms.warren or ca.los-angeles) */
```

Comment:
Switching this to lower-case kebab-case for consistency with the other custom cert fields